house/views.py

```diff
@@ -0,0 +1,26 @@
+from django.http import HttpResponse
+
+
+def home_page(request):
+    return HttpResponse("home page")
+
+
+def global_page(request):
+    filter_filled_1 = request.GET.get('filter_filled_1', '')
```

Comment:
So I understand - what does 'filter_filled_1' mean? 
Is there an option to change it to a meaningful name?
@ronyyosef

Reply:
Yes, sure!
That's the params that we are getting for the QueryParams
We can change it to whatever we want and send it from the UI in the same format

Reply:
don't think we changed it to meaningful name

Reply:
I suggest to delete or clear all unused methods in this file because the placeholder names and the implementation is not getting approved. and the "we will change it later" doesn't work here for merging by our mentors.
I have some of the methods already implemented Ill make another PR for that.
for now lets remove all unused methods or just write in all functions : `raise NotImplementedError` just so this will get merged. the PR is sitting for almost a week and its the base for all future PR using the urls and thats a big problem.

Reply:
I will do that as Lior suggests